### DISCUSSION TITLE
Need help clearing Change Log entries

### DISCUSSION THREAD

**Discussion by MrgampRus** [netbox-community/netbox#12120] [Help Wanted!]
Good afternoon. Please tell me if I can delete Change Log entries for a certain period. The fact is that I have configured a script for polling vcenter and due to the large number of virtual machines, a lot of Change Log entries appear.

**Comment by Domoninic** [comment]
In nbshell you can use ObjectChange.objects.all().delete() to delete all change logs. You would need to filter with ObjectChange.objects.filter() and filter on date to do a certain period. See https://docs.netbox.dev/en/stable/administration/netbox-shell/ for how to filter
